api.planx.uk/modules/pay/service/inviteToPay/createPaymentSendEvents.test.ts

```diff
@@ -77,7 +77,7 @@ describe("Create payment send events webhook", () => {
   it("returns a 400 if a required value is missing", async () => {
     await supertest(app)
       .post(ENDPOINT)
-      .set({ Authorization: process.env.HASURA_PLANX_API_KEY })
+      .set({ Authorization: process.env.HASURA_PLANX_API_KEY! })
```

Comment:
This is a type update that must have been picked up on a minor version bump of `supertest`